app/services/flow/flows/provider_merits.rb

```diff
@@ -14,11 +14,11 @@ class ProviderMerits < FlowSteps
         },
         respondents: {
           path: ->(application) { urls.providers_legal_aid_application_respondent_path(application) },
-          forward: :client_received_legal_helps,
+          forward: :statement_of_cases,
           check_answers: :check_merits_answers
         },
         client_received_legal_helps: {
-          path: ->(application) { urls.providers_legal_aid_application_client_received_legal_help_path(application) },
+          # path: ->(application) { urls.providers_legal_aid_application_client_received_legal_help_path(application) },
```

Comment:
An entry in flows should always have a path. I don't think this should be commented out.

Reply:
I had to comment it out because the code coverage was less than 100% for this. There's no rspec test for this because that page is not used within the app, however, it may be used in the future ( after speaking to the interaction designer).

Reply:
In general, it not a good idea to comment sections of codes out, because they may be used later. It seems like a good idea, but almost always just leads to lots of commented code being left if the code base.

If you are absolutely sure this needs to be commented out - then comment out the whole lot (`client_received_legal_helps` key and its value) - just the `path` part of the value.

Reply:
You could exclude this line from the code coverage report: https://github.com/colszowka/simplecov#ignoringskipping-code